services/api/src/utils/middleware/cors.js

```diff
@@ -0,0 +1,33 @@
+const cors = require('@koa/cors');
+const config = require('@bedrockio/config');
+
+// Note: Whitelisting domains here for added security layer.
+// To simply reflect the Origin header for a CORS response this
+// option may be removed, however this opens up the API to any
+// origin.
+//
+// Also note that although modern browsers will not allow this
+// header to be overridden it is still possible to spoof.
+
+const ALLOWED_ORIGINS = [config.get('APP_URL')];
+
+const DEFAULTS = {
+  origin: (ctx) => {
+    const origin = ctx.get('Origin');
+    if (ALLOWED_ORIGINS.includes(origin)) {
```

Comment:
I'm not sure if we want this restriction by default. One major selling point is that by using Bedrock you have an API that can be used by third parties. Let's discuss

Reply:
I'm totally fine with that. I can move this into a config that we can switch on/off with off as the default.